config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -242,6 +242,46 @@ presubmits:
           memory: "9000Mi"
           # during the tests more like 3-20m is used
           cpu: 2000m
+  - name: pull-kubernetes-csi-csi-driver-host-path-1-17-on-kubernetes-master
```

Comment:
It looks like you updated and ran the script, but then only committed one of the new jobs. The `dind_image` variable in the script also needs to be updated, otherwise we downgrade the images in the existing jobs.

Please add this  change, then re-run the script and commit all of the changes:
```
diff --git a/config/jobs/kubernetes-csi/gen-jobs.sh b/config/jobs/kubernetes-csi/gen-jobs.sh
index 2328891c9..72c3b9560 100755
--- a/config/jobs/kubernetes-csi/gen-jobs.sh
+++ b/config/jobs/kubernetes-csi/gen-jobs.sh
@@ -25,7 +25,7 @@ latest_stable_k8s_version="1.16.2"
 latest_stable_k8s_minor_version="1.16"
 
 # We need this image because it has Docker in Docker and go.
-dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191031-df9cbb4-master"
+dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191112-9f04410-master"
 
 # All kubernetes-csi repos which are part of the hostpath driver example.
 # For these repos we generate the full test matrix. For each entry here
```

Reply:
Ah I see. I didn't run the script originally - just copied another "k8s X on master" test and updated it for 1.17. 

Just updated and ran it, and committed the changes. Thanks!